src/parser/monk/mistweaver/modules/shadowlands/legendaries/AncientTeachingsoftheMonastery.tsx

```diff
@@ -15,15 +15,15 @@ import DonutChart from 'interface/statistics/components/DonutChart';
 
 class AncientTeachingsoftheMonastery extends Analyzer {
 
-  damageSpellToHealing = new Map();
+  damageSpellToHealing: Map<number, number> = new Map();
```

Comment:
Here you only specify the type for TS, but in the other places you also give the Map constructor the generic parameters; should pick one style :p

Reply:
I pick the style that annoys you the most :P so both at the same time <3